modules/mysql/outputs.tf

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Master
+output "master_instance_first_ip_address" {
+  value       = "${google_sql_database_instance.master.ip_address}"
+  description = "The first IPv4 address of the addesses assigned for the master instance"
+}
+
+output "master_instance_connection_name" {
+  value       = "${google_sql_database_instance.master.connection_name}"
+  description = "The connection name of the master instance to be used in connection strings"
+}
+
+output "master_instance_self_link" {
+  value       = "${google_sql_database_instance.master.self_link}"
+  description = "The URI of the master instance"
+}
+
+output "master_instance_server_ca_cert" {
+  value       = "${google_sql_database_instance.master.server_ca_cert}"
+  description = "The CA certificate information used to connect to the SQL instance via SSL"
+}
+
+output "master_instance_service_account_email_address" {
+  value       = "${google_sql_database_instance.master.service_account_email_address}"
+  description = "The service account email address assigned to the master instance"
+}
+
+// Replicas
+output "replicas_instance_first_ip_addresses" {
```

Comment:
Why is this the first instance? The splat syntax will fetch all of them.

Reply:
Fixed in: 0982e59ce16aa38c46893e0d0a7b786c774ee97b